### PR TITLE
data.propertyIsEnumerable causes error when override/not-exist

### PR DESCRIPTION
**Pull request by alcovescape**
use Object.prototype.propertyIsEnumerable which is the Object's original method regardless of data.

### CODE REVIEW

**Comment by xyc** [comment]
What use case is this? MDN says "Every object has a propertyIsEnumerable method." https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/propertyIsEnumerable
We don't intend to support non-standard behavior anyway.

**Comment by alcovescape** [comment]
you mis-interpreted MDN's statement, it has the **method**, but not the **property**.

Some code writes `x = Object.create(null) // over x = {}, the same-ish.`, 
which is not an uncommon (hyper-)optimisation to avoid all the proto chain (toString,...) appear in node.js source, vue (whether it runs faster or eventually lower memory, just don't bother.) 

they don't have `propertyIsEnumerable`, 
and entirely breaks the entire render with a single error..

It is an *object-inspector*, it should not ignore edge cases.

I will do a code search to find each method that breaks, and resubmit the PR.

**Comment by xyc** [comment]
Yeah, that makes sense and they are valid use cases. Thanks for bringing this up! Would you mind adding some cases in the storybook / test cases for this case? For example: `Object.create(null)` and `Object.assign(Object.create(null), {key: 'value'})`

`Object.create(null)` also shows "No properties" when you expand it in DevTools, but we can take care of that later.

File: src/object-inspector/ObjectInspector.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ const createIterator = (showNonenumerable, sortObjectKeys) => {
       }
 
       for (let propertyName of keys) {
-        if (data.propertyIsEnumerable(propertyName)) {
+        if (!Object.prototype.propertyIsEnumerable.call(data, propertyName)) {
```
Comment (xyc) [inline src/object-inspector/ObjectInspector.js:42]: we don't want to negate?

**Comment by xyc** [comment]
closed as fixed in #70

**Pull request closed without merging**